components/Community.jsx

```diff
@@ -221,7 +221,7 @@ const Form = () => {
         <div className='w-full flex justify-end'>
           <button
             type='submit'
-            className='mt-12 w-full md:w-36 rounded-md py-3 capitalize bg-[#5A75AB] text-white font-bold tracking-wider'>
+            className='mt-12 w-full md:w-36 rounded-md shadow-md hover:shadow-xl py-3 capitalize bg-[#5A75AB] text-white font-bold tracking-wider'>
```

Comment:
Looks good...this helps us keep consistency with other buttons such as those in navbar